src/server/pfs/server/testing/server_new_storage_test.go

```diff
@@ -35,6 +37,7 @@ type loadConfig struct {
 func newLoadConfig(opts ...loadConfigOption) *loadConfig {
 	config := &loadConfig{}
 	config.pachdConfig = newPachdConfig()
+	config.pachdConfig.StorageCompactionMaxFanIn = 2
```

Comment:
It seems like if this should be the default it would be in the `newPachdConfig` function, but this seems like a pretty low default.

Reply:
It should be 2 here because its the best number for the low memory testing, and least likely to explode CI.  The best default depends on the amount of memory we want to use. I think larger will almost always be better as far as time is concerned.

Reply:
I think it makes sense for the default to be a bit higher, and for the low memory testing you should configure it as two with a functional option for that field. Also, the default should be set with the other defaults in `newPachdConfig`.